pkg/controller/versions/strategy_v2_1.go

```diff
@@ -30,7 +30,7 @@ var (
 	v2_1ChartMapping = map[string]chartRenderingDetails{
 		DiscoveryChart: {
 			path:         "istio-control/istio-discovery",
-			enabledField: "",
+			enabledField: "istioDiscovery",
```

Comment:
why not `pilot.enabled`? isn't that the field upstream uses?

Reply:
We can use that if it seems more convenient to others. I chose this because it is the same as the chart's name and I wanted to express that this is for the whole chart and not just for the pilot deployment